docs/database/hbase.md

```diff
@@ -17,9 +17,12 @@ described on this page.
 <!-- semconv db.hbase(full,tag=tech-specific) -->
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`db.name`](../attributes-registry/db.md) | string | The HBase namespace. [1] | `mynamespace` | `Conditionally Required` If applicable. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.collection.name`](../attributes-registry/db.md) | string | The HBase table name. [1] | `mytable`; `ns:table` | `Conditionally Required` If applicable. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.collection.namespace`](../attributes-registry/db.md) | string | The HBase namespace. [2] | `mynamespace` | `Conditionally Required` If applicable and is not default. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Open question: hbase has default namespace, should we set to `default` or not set it?

Reply:
I'd suggest capturing it to avoid the question of whether missing value means default or not captured